public/css/take-exercise.css

```diff
@@ -0,0 +1,66 @@
+img{
+    height: 4rem;
+}
+
+.title{
+    text-align: center;
+    margin-top: 1rem;
+}
+
+.answering-list {
+    padding: 0;
+
+}
+
+@media (min-width: 40rem) {
+.row{
+    flex-direction: row;
+    margin-left: -1.0rem;
+    width: calc(95% - 100px);
+    padding : 0 1.0rem;
+}
+}
+.row {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 2rem;   
+    text-decoration: none;
+    width: 100%;
+}
+.card{
+    background-color: #eee; 
+    border-radius: .4rem;
+    text-align: center;
+}
+
+a{
+    display: block;
+}
+
+.button {
+    border-radius: 0.4rem;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+    text-decoration: none;
+    color: #ffffff;
+    background-color: #9b4dca;
+    text-transform: uppercase;
+    white-space: nowrap;
+    margin: 0.75rem;
+    font-size: 0.9rem;
+    font-weight: 700;
+    height: 10%;
```

Comment:
Le `height: 10%` est inutile, si tu le mets l'espace au dessus du Take it et plus grand qu'en bas.
> Avec le `height: 10%` :
> ![image](https://github.com/user-attachments/assets/2fec3f81-6a7b-4002-8c2a-ea77fa5bc122)

> Sans le `height: 10%`
> ![image](https://github.com/user-attachments/assets/1dea335a-fb84-4ca3-86a1-3404d3e16a72)